src/scala/io/bazel/rules_scala/target_split/BUILD

```diff
@@ -0,0 +1,25 @@
+load(":sculpt.bzl", "sculpt_json")
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_binary")
+
+sculpt_json(
+    name = "example",
+    srcs = ["File1.scala", "File2.scala"],
+    deps = [
+      "@io_bazel_rules_scala_scala_library",
+      "@io_bazel_rules_scala_scala_compiler",
+      "@io_bazel_rules_scala_scala_reflect",
+    ],
+    )
+
+java_binary(
+    name = "scalac_runner",
```

Comment:
Just a thought maybe this should be part of sculpt external repository since it’s a runner for that?